### ISSUE TITLE
Add method in tutorial to transition from loading in raven annotations to using them as training data

### ISSUE THREAD

**Issue by lydiakatsis** [kitzeslab/opensoundscape#730]
No split and save methods in 0.9.0 which makes it difficult for a first time user to understand how to integrate the raven annotations and data into the training data for the model. Couple of lines of script showing how to save sound clips and annotation table would be good.

**Comment by sammlapp** [comment]
Here's the basic mock-up of what you need to do to train on Raven-annotated audio:

Let's assume you have two lists: a list of `raven_file_paths` with one path per raven annotation file, and a list of the audio files `audio_file_paths` corresponding to each of the raven annotation files. 

First, we create labels on fixed-length audio clips. Let's say that we want to train our CNN on 2-second clips

```
from sklearn.model_selection import train_test_split
from opensoundscape import BoxedAnnotations

all_annotations = BoxedAnnotations.from_raven_files(raven_file_paths,audio_file_paths)

truth_df = all_annotations.one_hot_clip_labels(
    clip_duration=2,
    clip_overlap=0,
    min_label_overlap=0.25)

#split the labels into training and validation sets
train_df, valid_df = train_test_split(labels, test_size=0.3)
```

Now, let's train a CNN on these labels. We don't need to split up the audio files into clips, we can just pass CNN.train() the dataframes we created. The `classes` of our CNN should match columns of train_df and valid_df. 
```
from opensoundscape import CNN
class_list = ['class1','class2']
model = CNN(architecture='resnet18',sample_duration=2,classes=class_list)
model.train(train_df[class_list], validation_df[class_list], epochs=20, num_workers=8, batch_size=256)
```

**Comment by rhine3** [comment]
Now added to the basic CNN training tutorial by 09e7b005742f60a9b4fbf435afd8b0da3e92717b
